.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -145,7 +145,7 @@ runs:
       flavor: |
         latest=auto
```

Comment:
Should this also be set to the following?

```
latest=false
```

Reply:
Nah, so we'll keep publishing a latest tag in ECR, could still have some use anyway. Unless you feel strongly about removing latest altogether